common/buildings/wc_cultural_tradepost.txt

```diff
@@ -0,0 +1,314 @@
+		desc = tp_empty
+		trigger = {
+			FROM = { culture_group = harpy_group }
+			TECH_TRADE_PRACTICES = 3			
+		}
+		gold_cost = 350
+		build_time = 700
+		ai_creation_factor = 95
+		
+		cultural_techpoints = 0.02	
+		tradevalue = 20
+		trade_route_wealth = 2	
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Dwarfish Group
+	tp_dwarfish_stonemasons = {
+		potential = { in_major_trade_route_trigger = yes }
+		desc = tp_empty
+		trigger = {
+			FROM = { culture_group = harpy_group }
+			TECH_TRADE_PRACTICES = 3			
+		}
+		gold_cost = 350
+		build_time = 700
+		ai_creation_factor = 95
+
+		garrison_size = 1.5		
+		retinuesize = 10
+		military_techpoints = 0.01	
+		tradevalue = 20
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Kobolt Group
+	tp_chandlers_guild = {
+		potential = { in_major_trade_route_trigger = yes }
+		desc = tp_empty
+		trigger = {
+			FROM = { culture_group = harpy_group }
+			TECH_TRADE_PRACTICES = 3			
+		}
+		gold_cost = 300
+		build_time = 550
+		ai_creation_factor = 95
+
+		monthly_character_piety = 0.01
+		economic_techpoints = 0.01
+		cultural_techpoints = 0.01
+		tradevalue = 20
+		trade_route_wealth = 2			
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Goblins Group
+	tp_merchant_bots = {
+		potential = { in_major_trade_route_trigger = yes }
+		desc = tp_empty
+		trigger = {
+			FROM = { culture_group = gnome_group }
+			TECH_TRADE_PRACTICES = 3			
+		}
+		gold_cost = 200
+		build_time = 400
+		ai_creation_factor = 95
+
+		retinuesize = 10
+		military_techpoints = 0.01
+		economic_techpoints = 0.01
+		cultural_techpoints = 0.01
+		tradevalue = 10		
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Murloc Group
+	tp_murloc_coralmasons = {
+		potential = { in_major_trade_route_trigger = yes }
+		desc = tp_empty
+		trigger = {
+			FROM = { culture_group = murloc_group }
+			TECH_TRADE_PRACTICES = 3			
+		}
+		gold_cost = 450
+		build_time = 700
+		ai_creation_factor = 95
+
+		tech_growth_modifier = 0.2
+		cultural_techpoints = 0.01
+		tradevalue = 30		
+		trade_route_wealth = 1
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	#######Religious################
+	
+	##Arcane religion
+	tp_portal_center = {
+		potential = { in_major_trade_route_trigger = yes }
+		desc = tp_empty
+		trigger = {
+			FROM = { religion = arcane_religion }
+			TECH_TRADE_PRACTICES = 3			
+		}
+		gold_cost = 350
+		build_time = 700
+		ai_creation_factor = 95
+
+		monthly_character_piety = 0.2
+		tech_growth_modifier = 0.3
+		military_techpoints = 0.02
+		economic_techpoints = 0.02
+		cultural_techpoints = 0.02
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}	
+	
+	##Altar of Wealth
+	tp_altar_of_wealth = {
+		potential = { in_major_trade_route_trigger = yes }
+		desc = tp_empty
+		trigger = {
+			FROM = { religion = arcane_religion }

```

Comment:
`arcane_religion` has 2 buildings?

Reply:
arcane building is basicaly related to travel and tech and provides no direct economy boost. This building was made with idea in mind first then name and theme